src/site/stages/build/process-cms-exports/schemas/output/media-image.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    entityType: { type: 'string', enum: ['media'] },
+    entityBundle: { type: 'string', enum: ['image'] },
+    name: { type: 'string' },
+    thumbnail: { $ref: 'output/file' },
+    created: { type: 'number' },
+    changed: { type: 'number' },
```

Comment:
Are these also present in the corresponding GraphQL query result? I ask only because they're ignored in many other bundles.

Reply:
They actually aren't. I'll remove them 👍